src/host/ft_fuzzer/Host.FuzzWrapper.vcxproj

```diff
@@ -78,7 +78,7 @@
     <!-- In theory, we may want to build with a normal main() when Fuzzing is not enabled. -->
     <!-- So, let's only add the fuzzer to the link line when we're building for Fuzzing. -->
     <Link>
-      <AdditionalDependencies>clang_rt.fuzzer-$(OCClangArchitectureName).lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>WinMM.Lib;clang_rt.fuzzer-$(OCClangArchitectureName).lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Wut. Why was WinMM suddenly needed? Play sound?

Reply:
Yeah, no idea why this came up:
https://github.com/microsoft/terminal/blob/848e353b9cc19d6a3c114ee6261de595d94f19c7/src/interactivity/win32/windowproc.cpp#L675